pandas/core/series.py

```diff
@@ -2076,8 +2076,7 @@ def idxmin(self, axis=0, skipna=True, *args, **kwargs):
         >>> s.idxmin(skipna=False)
         nan
         """
-        skipna = nv.validate_argmin_with_skipna(skipna, args, kwargs)
```

Comment:
no longer validating args/kwargs?

Reply:
sorry missed this comment

cc @tonyyyyip

Reply:
Right. I think if we let `idxmax` call `self.argmax(axis=axis, skipna=skipna, *args, **kwargs)` then the args and kwargs can be validated by argmax's validator. I can make another PR if this is desirable.

Reply:
that would be great @tonyyyyip